modules/platforms/dotnet/Apache.Ignite.Core/Cache/Configuration/CacheConfiguration.cs

```diff
@@ -948,5 +971,12 @@ public string MemoryPolicyName
         /// </summary>
         [IgniteExperimental]
         public PlatformCacheConfiguration PlatformCacheConfiguration { get; set; }
+
+        /// <summary>
+        /// Cluster node filter.
```

Comment:
`Gets or sets the cluster node filter. Cache will be started only on nodes that match the filter.`